src/components/BookList.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Book from './Book';
+import CreateBook from './CreateBook';
+
+function BookList() {
+  const books = useSelector((state) => state.books);
+
+  return (
+    <ul className="book-container">
+      {books.map((book) => (
+        <li className="Main-content" key={book.id}>
+          <Book title={book.title} author={book.author} />
+          <CreateBook />
+        </li>
```

Comment:
- [ ] The CreateBook component should be rendered outside the loop that renders the individual Book component. Currently, it is being rendered every time a single Book component is rendered and that means you have many forms instead of one based on the design. 👍 

![image](https://user-images.githubusercontent.com/77106962/214692791-9162d8c7-f217-4336-adcb-a35198423a39.png)

Reply:
Thanks for noticing the glitch I've fixed it as per requested